modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/actions/PreIssueAccessTokenPasswordGrantTestCase.java

```diff
@@ -149,7 +179,7 @@ public class PreIssueAccessTokenPasswordGrantTestCase extends ActionsBaseTestCas
     @BeforeClass(alwaysRun = true)
     public void testInit() throws Exception {
 
-        super.init(TestUserMode.TENANT_USER);
+        super.init(userMode);
```

Comment:
Let's move this to the constructor.

Reply:
We should not move this to the constructor. Constructor just instantiates the test class. This is the exact test case init.